Sources/Tuvi.Core.Impl/Utils/Keys/NetworkPublicKeyRules.cs

```diff
@@ -0,0 +1,97 @@
+﻿// ---------------------------------------------------------------------------- //
+//                                                                              //
+//   Copyright 2025 Eppie (https://eppie.io)                                    //
+//                                                                              //
+//   Licensed under the Apache License, Version 2.0 (the "License"),            //
+//   you may not use this file except in compliance with the License.           //
+//   You may obtain a copy of the License at                                    //
+//                                                                              //
+//       http://www.apache.org/licenses/LICENSE-2.0                             //
+//                                                                              //
+//   Unless required by applicable law or agreed to in writing, software        //
+//   distributed under the License is distributed on an "AS IS" BASIS,          //
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   //
+//   See the License for the specific language governing permissions and        //
+//   limitations under the License.                                             //
+//                                                                              //
+// ---------------------------------------------------------------------------- //
+
+using System;
+using Tuvi.Core.Entities;
+
+namespace Tuvi.Core.Utils
+{
+    internal interface INetworkPublicKeyRules
+    {
+        bool IsSyntacticallyValid(string value);
+        bool TrySemanticValidate(string value);
+        bool TryValidate(string value);
```

Comment:
I think the prefix **Try** is unnecessary here

Reply:
I will rename it in the next PR.

Reply:
Add ToDo: